### PR TITLE
DDF-2479 Removed WFS Endpoint

### PR DESCRIPTION
**Pull request by emmberk**
#### What does this PR do?
This PR removes the WFS Endpoint.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@glenhein @peterhuffer @troymohl 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@millerw8
@kcwire
#### How should this be tested?
Confirm tests/itests succeed.
#### Any background context you want to provide?
The WFS endpoint is no longer supported. Confirm that the other spatial capabilities still work.
#### What are the relevant tickets?
https://codice.atlassian.net/browse/DDF-2479
#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by glenhein** [comment]
lgtm, however I don't know the ramifications of removing all that code, so I assume it's the correct action

**Comment by kcwire** [comment]
👍  - @emmberk as a double check to ensure there are no lingering dependencies on wfs-endpoint, you could run `mvn -DdryRun=true release:prepare`.

**Comment by peterhuffer** [comment]
👍

**Comment by beyelerb** [comment]
👍 lgtm - did some searching around and didn't see any other references anywhere - would be interested to see the result of the release:prepare that @kcwire mentioned.

**Comment by emmberk** [comment]
Successfully completed build with `mvn -DdryRun=true release:prepare`

File: distribution/docs/src/main/resources/_contents/_spatial-contents/integrating-spatial-contents.adoc [hunk 1]
```diff
@@ -3598,7 +3598,7 @@ ${branding} supports the ability to integrate WFS 1.0 and WFS 2.0 Web Services.
 
 [NOTE]
 ====
-${branding} does not include a supported WFS Web Service (Endpoint) implementation.
+${branding} does not include a WFS Web Service (Endpoint) implementation.
```
Comment (millerw8) [inline distribution/docs/src/main/resources/_contents/_spatial-contents/integrating-spatial-contents.adoc]: I know it didn't happen here... but I don't like Endpoint being in ()s.   I always think of things in ()s as things tha could be removed without changing the meaning of the sentence... but without the Endpoint word... this thing is almost saying we support WFS and we don't support it.

**Comment by millerw8** [comment]
Built with clean repository, running all tests and building documentation.  Tested WFS Source (v1.0 and v2.0) and did a self federation over CSW 2.0.2 to confirm no hidden dependencies were broken inside the spatial app.

**Comment by bradh** [comment]
I would have considered WFS endpoint as public API, so this needs to be in DDF 3.0.

**Pull request merged**